packages/ui-extensions/src/surfaces/admin/api/standard/standard.ts

```diff
@@ -54,6 +55,11 @@ export interface StandardApi<ExtensionTarget extends AnyExtensionTarget> {
    */
   intents: Intents;
 
+  /**
+   * Provides methods for storing data in the extension.
```

Comment:
Nit... trying to be more specific about _which_ methods. Feel free to ignore. 😄 
```suggestion
   * Provides methods for setting, getting, and clearing browser data in from the extension
```